packages/shared/src/components/PostOptionsReadingHistoryMenu.tsx

```diff
@@ -114,34 +114,35 @@ export default function PostOptionsReadingHistoryMenu({
     });
   };
 
+  const options = [
+    {
+      icon: <MenuIcon Icon={ShareIcon} />,
+      label: 'Share post via...',
+      action: onShare,
+    },
+    {
+      icon: getBookmarkIconAndMenuIcon(post?.bookmarked),
+      label: post?.bookmarked ? 'Remove from bookmarks' : 'Save to bookmarks',
+      action: onToggleBookmark,
+    },
+    {
+      icon: <MenuIcon Icon={XIcon} />,
+      label: 'Remove post',
+      action: () => onHideHistoryPost(post?.id),
+    },
+  ];
+
   return (
     <>
```

Comment:
Non-blocking. Can you maybe remove the extra Fragment? I understand this was a legacy thing, just for cleanup 😄